### DISCUSSION TITLE
Where does opensnitch store it's rules?

### DISCUSSION THREAD

**Discussion by DigiAngel** [evilsocket/opensnitch#1046] [General]
Great app thank you for it.  I've recently swapped drives and installed Ubuntu 22...I have the saved drive.  I had a large amount of firewall rules and I was wondering where those rules were stored at.  I looked in .config/opensnitch, but that settings file seemed pretty small.  Thank you!

**Comment by izar3** [comment]
For arch its /etc/opensnitchd/ if not try searching in your file manager for opensnitch

but its best to use import & export rules on the UI

**Comment by DigiAngel** [comment]
Just what I needed..thank you much!
